crypto/fipsmodule/service_indicator/service_indicator_test.cc

```diff
@@ -2006,6 +2006,13 @@ TEST(ServiceIndicatorTest, DRBG) {
   CTR_DRBG_clear(&drbg);
 }
 
+// Verifies that the openssl_version_string is as expected.
+// Since this is running in FIPS mode it should end in FIPS
+// Update this when the OpenSSL version number is modified
+TEST(ServiceIndicatorTest, OpenSSLVersionString) {
+  ASSERT_STREQ(openssl_version_string(), "OpenSSL 1.1.1 (compatible; BoringSSL) FIPS");
```

Comment:
boringssl?

Reply:
> boringssl?

That's what we have in our OPENSSL_VERSION_TEXT at the moment. I could change it though.

Reply:
We should define something AWS-LC specific. It is confusing to keep using OpenSSL and BoringSSL names IMO.

There are no dependencies on this atm, so we can make a good choice up-front disconnecting from upstream.

Probably need to keep `OPENSSL_VERSION_TEXT` for compatibility though.

Reply:
If we want an `AWSLC_VERSION_TEXT` should we stick with the same version number we're using for `OPENSSL_VERSION_TEXT` or should we change it to something else? We could pick our own numbering scheme but that might cause more confusion.